lib/checks/aria/aria-hidden-body.js

```diff
@@ -0,0 +1,11 @@
+var aria = /^aria-hidden/;
+if (node && node.hasAttributes()) {
```

Comment:
Can you check for the attribute with `node.hasAttribute()` to avoid having to iterate?